Gemfile

```diff
@@ -14,6 +14,6 @@ gem "rack-test", "~> 2.0"
 gem "redcarpet", "~> 3.0"
 gem "redis", ">= 4.0", "< 6"
 gem "rspec", "~> 3.0"
-gem "sinatra"
+gem "toys", "~>0.15"
```

Comment:
Toys should generally not be included in gemfiles. (For example, you just `toys test` instead of `bundle exec toys test`.) Instead, Toys sets up the bundle itself. This is because it might not necessarily know _which_ bundle to use until it parses its arguments and determines which tool you're running.

Reply:
Ack, fixed